gui/src/app/components/entity-details/content/content.js

```diff
@@ -256,14 +256,17 @@ class EntityContent extends React.Component {
   */
   _handleTagClick(e) {
     e.stopPropagation();
+    const tag = e.target.getAttribute('data-id');
     this.props.changeState({
       hash: null,
       search: {
-        tags: e.target.getAttribute('data-id'),
+        tags: tag,
         text: ''
       },
       store: null
     });
+    this.props.analytics.addCategory('Tag').sendEvent(
+      this.props.analytics.CLICK, {label: `tag: ${tag}`});
```

Comment:
Don't think we need the prefix of tag: when setting it to the tag category.

Reply:
I've been adding the prefix because there are also some global details appended to the label. The result in this case would be something like: `model committed: false, user authenticated: true, tag: ops1`. The tag prefix would help identify that bit of data.